tests/infra/remote.py

```diff
@@ -391,6 +415,20 @@ def start(self, timeout=10):
             stderr=self.stderr,
             env=self.env,
         )
+        self.suspension_proc = psutil.Process(pid=self.proc.pid)
+
+    def suspend(self):
+        try:
+            self.suspension_proc.suspend()
+            LOG.info(f"Node {self.name} suspended...")
+            return True
+        except psutil.NoSuchProcess:
+            LOG.info(f"Node {self.name} can not be suspended...")
+            return False
+
+    def resume(self):
+        self.suspension_proc.resume()
+        LOG.info(f"Node {self.name} resuming from suspension...")
```

Comment:
Why do we wrap suspend into a try/catch but not resume? I'd rather let the execption bubble up in both cases.